.travis.yml

```diff
@@ -2,12 +2,14 @@ language: go
 
 go:
   - "1.12.x"
+  - "1.13.x"
```

Comment:
should we update the go mod file? Also is there any reason to not choose `1.15`?

Reply:
if we upgrade `go.mod` version to `1.14` for example then it will be only able to compile with versions >- 1.14 so its not backward compatible. for 1.15 not any reason I just added a few extra targets